### PR TITLE
Update WooCommerce Blocks to 9.1.3

### PR DESCRIPTION
**Pull request by gigitux**
This pull updates the WooCommerce Blocks plugin to 9.1.3. 
## Blocks 9.1.2

* [Release PR](https://github.com/woocommerce/woocommerce-blocks/pull/8004)
* [Testing instructions](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/testing/releases/912.md)

## Blocks 9.1.3

* [Release PR](https://github.com/woocommerce/woocommerce-blocks/pull/8011)
* [Testing instructions](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/testing/releases/913.md)

### CODE REVIEW

**Review by Aljullu — DISMISSED** [review]
Tested and I confirm the Products block is available. :+1:

**Comment by gigitux** [comment]
We noticed some issues with the other two blocks. We are working to do another release.

**Review by imanish003 — DISMISSED** [review]
Tested & confirm that all `products block` patterns are working fine.

File: plugins/woocommerce/composer.lock [hunk 1]
```diff
@@ -1748,16 +1748,16 @@
...
+                "reference": "1dc7ceb4a24aede938c7af2a9ed1de09609ca770"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/sebastianbergmann/comparator/zipball/1071dfcef776a57013124ff35e1fc41ccd294758",
-                "reference": "1071dfcef776a57013124ff35e1fc41ccd294758",
+                "url": "https://api.github.com/repos/sebastianbergmann/comparator/zipball/1dc7ceb4a24aede938c7af2a9ed1de09609ca770",
+                "reference": "1dc7ceb4a24aede938c7af2a9ed1de09609ca770",
                 "shasum": ""
             },
```
Comment (dinhtungdu) [inline plugins/woocommerce/composer.lock]: @psealock This causes the Unit tests to fail. Reverting this change locally, (make sure to run `composer i` after that) and the test is passing for me.

image

Comment (imanish003) [reply plugins/woocommerce/composer.lock]: I also reverted this change locally & confirm that it fixes the failing test.

File: plugins/woocommerce/composer.json [hunk 2]
```diff
@@ -21,13 +21,14 @@
...
-		"woocommerce/woocommerce-blocks": "9.1.1"
+		"woocommerce/woocommerce-blocks": "9.1.3"
 	},
 	"require-dev": {
 		"bamarni/composer-bin-plugin": "^1.4",
 		"yoast/phpunit-polyfills": "^1.0",
 		"phpunit/phpunit": "7.5.20",
-		"automattic/jetpack-changelogger": "3.1.3"
+		"automattic/jetpack-changelogger": "3.1.3",
+		"sebastian/comparator": "3.0.3"
```
Comment (psealock) [inline plugins/woocommerce/composer.json:31]: Version 3.0.5 of `sebastian/comparator` was causing the testing oddities, so I've pinned to 3.0.3.

**Review by jonathansadowski — APPROVED** [review]

**Pull request merged**